geonetwork/config/config-security-georchestra.xml

```diff
@@ -142,4 +142,11 @@
     <constructor-arg index="0" type="java.lang.String" name="userDnBase" value="${ldap.base.search.base}"/>
     <constructor-arg index="1" type="java.lang.String" name="usernameAttribute" value="${ldap.sync.user.search.attribute}"/>
   </bean>
+
+  <!-- redirect to cas when need to be logged in -->
+  <bean id="authenticationEntryPoint"
+    class="jeeves.config.springutil.JeevesNodeAwareLoginUrlEntryPoint">
+    <constructor-arg index="0" value="/?login" />
```

Comment:
Did you check the behavior with `?login` instead of "/?login" ? I'm curious

Reply:
`/?login`:

```
% curl -i 'http://localhost:8080/geonetwork/srv/fre/admin.console'
HTTP/1.1 302 Found
Cache-Control: no-cache, no-store, max-age=0, must-revalidate
Pragma: no-cache
Expires: Thu, 01 Jan 1970 00:00:00 GMT
X-XSS-Protection: 1; mode=block
X-Frame-Options: SAMEORIGIN
X-Content-Type-Options: nosniff
Set-Cookie: JSESSIONID=node0qutfqwbrekrrpkkha74lw2265.node0;Path=/
Expires: Thu, 01 Jan 1970 00:00:00 GMT
Server: Jetty(9.4.7.v20170914)
Location: http://localhost:8080/geonetwork/;jsessionid=node05ys6av95dwhk19w0hx0pn6h672.node0?login
```

`?login`: unable to launch the webapp:

```
geonetwork_1    | Caused by: 
geonetwork_1    | java.lang.IllegalArgumentException: loginFormUrl must be specified and must be a valid redirect URL
geonetwork_1    | 	at org.springframework.util.Assert.isTrue(Assert.java:65)
geonetwork_1    | 	at org.springframework.security.web.authentication.LoginUrlAuthenticationEntryPoint.afterPropertiesSet(LoginUrlAuthenticationEntryPoint.java:103)

```